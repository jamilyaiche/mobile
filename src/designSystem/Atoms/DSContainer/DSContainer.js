import React from 'react'
import { View } from 'react-native'
import Style from '../../../assets/styles/Style'

const DSContainer = (props) => {
  return (
        <View style={[Style.container,props.styleContainer]}>
            {props.children}
        </View>
  )
}

DSContainer.defaultProps = {
  styleContainer:{}
}

export default DSContainer
