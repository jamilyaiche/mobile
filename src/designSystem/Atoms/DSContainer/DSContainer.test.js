import React from 'react';
import renderer from 'react-test-renderer'
import DSContainer from "./DSContainer"
import {Text} from "react-native"

describe('container tests',()=>{
    it('test container snapshot',()=>{
        const wrapper=renderer.create(<DSContainer>test</DSContainer>)
        .toJSON()
        expect(wrapper).toMatchSnapshot()
    })

    it('test container children',()=>{
        const wrapper=renderer.create(<DSContainer><Text>text</Text></DSContainer>)
        expect(wrapper.root.findAllByType(Text).length===1)
        .toEqual(true)
    })
})