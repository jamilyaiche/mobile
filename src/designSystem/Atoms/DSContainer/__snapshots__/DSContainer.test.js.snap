// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container tests test container snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(255, 255, 255, 1)",
        "display": "flex",
        "flex": 1,
        "flexDirection": "column",
        "padding": 20,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  test
</View>
`;
